docs/docs/bloom-filter.md

```diff
@@ -71,57 +71,30 @@ Using Redis Stack's Bloom filter for this type of application provides these ben
 - Very fast and efficient way to do a common operation 
 - No need to invest in expensive infrastructure  
 
-## Examples:
+## Example
 
-* Adding new items to the filter
+Consider a bike manufacturer that makes a million different kinds of bikes and you'd like to avoid using a duplicate model name in new models. A bloom filter can be used to detect duplicates. In the example that follows,  you'll create a filter with space for a million entries and with a 0.1% error rate. Add one model name and check if it exists. Then add multiple model names and check if they exist.
 
-> A new filter is created for you if it does not yet exist
-
-```
-> BF.ADD newFilter foo
+{{< clients-example bf_tutorial bloom >}}
+> BF.RESERVE bikes:models 0.001 1000000
+OK
+> BF.ADD bikes:models "Smoky Mountain Striker"
 (integer) 1
-```
-
-* Checking if an item exists in the filter
-
-```
-> BF.EXISTS newFilter foo
+> BF.EXISTS bikes:models "Smoky Mountain Striker"
 (integer) 1
-```
-
-```
-> BF.EXISTS newFilter notpresent
-(integer) 0
-```
-
-* Adding and checking multiple items
-
-```
-> BF.MADD myFilter foo bar baz
+> BF.MADD bikes:models "Rocky Mountain Racer" "Cloudy City Cruiser" "Windy City Wippet"
 1) (integer) 1
 2) (integer) 1
 3) (integer) 1
-```
-
-```
-> BF.MEXISTS myFilter foo nonexist bar
+> BF.MEXISTS bikes:models "Rocky Mountain Racer" "Cloudy City Cruiser" "Windy City Wippet"
 1) (integer) 1
-2) (integer) 0
+2) (integer) 1
 3) (integer) 1
-```
-
-* Creating a new filter with custom properties
+{{< /clients-example >}}
 
-```
-> BF.RESERVE customFilter 0.0001 600000
-OK
-```
-
-```
-> BF.MADD customFilter foo bar baz
-```
+Note: there is always a chance that even with just a few items, there could be a collision, yielding a false positive.
```

Comment:
I'm not sure if 'collision' is clear here. You need to understand how a Bloom filter works to understand what collision means, and probably we don't want to explain it here. I would change it to "there could be a false positive" (and maybe this term should be explained here as well).

Reply:
@dwdougherty as well - how does "Note: there is always a chance that even with just a few items, there could be a false positive, meaning an item could "exist" even though it has not been explicitly added to the Bloom filter. For a more in depth understanding of the probabilistic nature of a Bloom filter, check out the blog posts linked at the bottom of this page."

Reply:
LGTM